php/kuiper/composer.json

```diff
@@ -0,0 +1,26 @@
+{
+  "require": {
+    "php": "^8.1",
+    "ext-json": "*",
+    "kuiper/web": "^0.8"
+  },
+  "require-dev": {
+    "kuiper/component-installer": "^0.1"
+  },
```

Comment:
do we need this ?

Reply:
not require at runtime, only for development

Reply:
ok, maybe use `@stable` then :-)